Index: Practice/Demo.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Practice;\r\n\r\nimport java.util.Arrays;\r\nimport java.util.Scanner;\r\n\r\nimport static java.lang.System.in;\r\n\r\npublic class Demo {\r\n    public static void main(String[] args) {\r\n        Scanner scanner = new Scanner(in);\r\n\r\n        int[] mainArray= Arrays.stream(scanner.nextLine().split(\" \")).mapToInt(Integer::parseInt).toArray();\r\n        int [] condensedArray=new int[mainArray.length-1];\r\n\r\n        if (mainArray.length==1){\r\n            System.out.println(mainArray[0]);\r\n            return;\r\n        }\r\n        for (int i = 0; i <mainArray.length ; i++) {\r\n            for (int j = 0; j <condensedArray.length-i ; j++) {\r\n                condensedArray[j]=mainArray[j]+mainArray[j+1];\r\n            }\r\n            mainArray=condensedArray;\r\n        }\r\n        System.out.println(condensedArray[0]);\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Practice/Demo.java b/Practice/Demo.java
--- a/Practice/Demo.java	(revision 7a238b54d14a750759dad083b87acf9f9bb66c27)
+++ b/Practice/Demo.java	(date 1666388675697)
@@ -9,19 +9,21 @@
     public static void main(String[] args) {
         Scanner scanner = new Scanner(in);
 
-        int[] mainArray= Arrays.stream(scanner.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();
-        int [] condensedArray=new int[mainArray.length-1];
+        int n = Integer.parseInt(scanner.nextLine());
+        int l = Integer.parseInt(scanner.nextLine());
 
-        if (mainArray.length==1){
-            System.out.println(mainArray[0]);
-            return;
-        }
-        for (int i = 0; i <mainArray.length ; i++) {
-            for (int j = 0; j <condensedArray.length-i ; j++) {
-                condensedArray[j]=mainArray[j]+mainArray[j+1];
+        for (int index1 = 1; index1 <=n ; index1++) {
+            for (int index2 = 1; index2 <=n ; index2++) {
+                for (char index3 = 97; index3 < 97 + l; index3++) {
+                    for (char index4 = 97; index4 <97+l ; index4++) {
+                        for (int index5 = 1; index5 <=n ; index5++) {
+                            if (index5>index1&&index5>index2) {
+                                System.out.printf("%d%d%s%s%d ", index1, index2, index3, index4, index5);
+                            }
+                        }
+                    }
+                }
             }
-            mainArray=condensedArray;
         }
-        System.out.println(condensedArray[0]);
     }
 }
Index: ObjectsandClassesLab/P01RandomizeWords.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ObjectsandClassesLab/P01RandomizeWords.java b/ObjectsandClassesLab/P01RandomizeWords.java
new file mode 100644
--- /dev/null	(date 1667312435776)
+++ b/ObjectsandClassesLab/P01RandomizeWords.java	(date 1667312435776)
@@ -0,0 +1,30 @@
+package ObjectsandClassesLab;
+
+import java.util.Random;
+import java.util.Scanner;
+
+import static java.lang.System.in;
+
+public class P01RandomizeWords {
+    public static void main(String[] args) {
+        
+    Scanner scanner = new Scanner(in);
+
+    String input = scanner.nextLine();
+
+    String [] wordArr= input.split("\\s+");
+
+    Random rndGenerator = new Random();
+        for (int i = 0; i < wordArr.length ; i++) {
+            int indexX = rndGenerator.nextInt(wordArr.length);
+            int indexY = rndGenerator.nextInt(wordArr.length);
+
+            String oldWordX = wordArr[indexX];
+            wordArr[indexX] = wordArr[indexY];
+            wordArr[indexY] = oldWordX;
+        }
+        System.out.println(String.join(System.lineSeparator(),wordArr));
+    }
+    
+    
+}
Index: MidExamPreparation/ComputerStore.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/MidExamPreparation/ComputerStore.java b/MidExamPreparation/ComputerStore.java
--- a/MidExamPreparation/ComputerStore.java	(revision 7a238b54d14a750759dad083b87acf9f9bb66c27)
+++ b/MidExamPreparation/ComputerStore.java	(date 1666390539059)
@@ -1,0 +1,39 @@
+package MidExamPreparation;
+import java.util.Scanner;
+
+import static java.lang.System.in;
+
+public class ComputerStore {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(in);
+
+        String pricesWithoutVAT = scanner.nextLine();
+        double sum =0;
+        while (!pricesWithoutVAT.equals("special")){
+            if (pricesWithoutVAT.equals("regular")){
+                break;
+            }
+            double priceWithoutVAT = Double.parseDouble(pricesWithoutVAT);
+            if (priceWithoutVAT>0){
+                sum+=priceWithoutVAT;
+            }else {
+                System.out.println("Invalid price!");
+            }
+            pricesWithoutVAT= scanner.nextLine();
+        }
+        double taxes = sum * 0.2;
+        double totalPrice= sum+taxes;
+        if (sum>0){
+            if (pricesWithoutVAT.equals("special")){
+                totalPrice-=totalPrice*0.1;
+            }
+            System.out.printf("Congratulations you've just bought a new computer!\n" +
+                    "Price without taxes: %.2f$\n" +
+                    "Taxes: %.2f$\n" +
+                    "-----------\n" +
+                    "Total price: %.2f$\n",sum,taxes,totalPrice);
+        }else{
+            System.out.println("Invalid order!");
+        }
+    }
+}
